README.md

```diff
@@ -37,3 +37,175 @@ docker run -it --rm \
   -v $(pwd)/boardswarm/share/server.conf:/etc/boardswarm/server.conf \
   ghcr.io/boardswarm/boardswarm:main
 ```
+
+## Manual Installation
+
+These instructions are for configuring a simple unencrypted setup with static
+JWKS based authentication.
+
+### Building
+
+Boardswarm needs rustc version 1.70 or newer, this rules out a number of stable
+distributions for now (the kind of thing you may want to run your
+infrastructure on). There are however a number of solutions
+
+ - Rather than installing the distribution version of Rust, use
+   [rustup](https://www.rust-lang.org/tools/install) to install the latest
+   version directly from the rust project.
+
+ - Build on a system that has the required version of rust installed and, as
+   rust builds static binaries, we can copy them to the target server once
+   built. (Note that rust will note the version of libc used during the build
+   and complain if the expected version isn't found, making this solution
+   unreliable at times.) For instance, assuming a Debian system:
+   ```
+   $ sudo apt install rustc
+   ```
+
+ - If installing software outside of the systems package management isn't for
+   you and building elsewhere isn't an option, then creating a container from
+   which to build boardswarm might be a solution. A simple Dockerfile to just
+   build boardswarm is as follows:
```

Comment:
I don't like having that untestable in the documentation; It just means it'll bitrot

Reply:
I'm not sure how to best resolve this. I was unable to build boardswarm on a Debian Stable machine that I'd got setup for my lab control. Building on Debian Testing worked, but then wouldn't run on the Stable machine due to different versions of libc being used. I doubt I'm the only one that prefers not to stuff a load of `curl | sh` installs on systems, especially those that I hope to keep running without complete reinstalls for extended periods. The existing Dockerfiles don't provide a means to access the binaries outside of Docker once they've been built.

Would having this as a Dockerfile in the root directory of the project be an option you'd consider?

Reply:
We have already two docker files from which you seem to have copied this from; if you split the builder stage into a first stage that creates the builder and second one that does the build then you can easily build just the build container from that if you prefer.

Also while you're right, especially at this point of development, i don't think the type of audience that might give boardswarm a go *and* wnats to build it themselves should have to be thought how top installer Rust per se :).

for those that don't want or cant themselves easily, but do run bookworm (or compatibl repos); that's why i'm doing https://github.com/sjoerdsimons/boardswarm/tree/dist to allow downloading regularily build tarballs (which i'd also want to use for my own deployments)